installer/client/src/components/DatabaseConfigDialog.vue

```diff
@@ -25,7 +25,7 @@
       ></oxd-classic-table>
       <oxd-check-box
         v-model="checked"
-        option-label="I want to continue upgrading the OrangeHRM system to version 5.6 and I am aware that by doing so, any gathered data in incomplete features/add-ons will be inaccessible."
+        option-label="I want to continue upgrading the OrangeHRM system to version 5.6.1 and I am aware that by doing so, any gathered data in incomplete features/add-ons will be inaccessible."
```

Comment:
any chance to remove this now. otherwise we have to change the version in every release